auth-groups.json

```diff
@@ -1,11 +1,12 @@
 {
   "staging": {
```

Comment:
This all looks good!

Really small thing but thought it might be nice to re-order the environments to be in order of their 'importance', maybe `dev`, `staging`, `production`, what do you think?

Reply:
Absolutely. Fixed 👍